draft-irtf-cfrg-hpke.md

```diff
@@ -264,42 +265,52 @@ values are two bytes long.
 
 ## DH-Based KEM
 
-Suppose we are given a Diffie-Hellman group that provides the
-following operations:
+Suppose we are given a Diffie-Hellman group, and a hash function,
+providing the following operations:
 
 - GenerateKeyPair(): Generate an ephemeral key pair `(sk, pk)`
   for the DH group in use
 - DH(sk, pk): Perform a non-interactive DH exchange using the
-  private key sk and public key pk to produce a shared secret
-  of length Npk
+  private key sk and public key pk to produce a Diffie-Hellman
+  shared secret of length Npk
```

Comment:
It seems like it might be worth defining a different length, say `Ndh` for this, since there are definitely schemes where the shared secret is not of length `Npk`.